src/api/queries.js

```diff
@@ -83,6 +83,11 @@ exports.getQuestionnaire = `
     }
   }
 
+  fragment metadataFragment on Metadata {
+    key
+    type
+  }
+
```

Comment:
Not sure the fragment adds much value here as it's not being re-used. It's your call, but I would just inline the fields.